Resources/Prototypes/Datasets/tips.yml

```
@@ -1,61 +1,61 @@
+  - "Как сотрудник службы безопасности, помните, что После не значит Вследствие. Возможно кто-то просто оказался не в том месте и не в то время!"
+  - "Будучи призраком, вы можете использовать меню взаимодействия, чтобы автоматически следовать и крутиться вокруг любого объекта в игре."
+  - "Будучи предателем, вам может быть поручено устранить другого предателя, а может быть и наоборот."
+  - "Будучи предателем, вы можете использовать гарнитуру синдиката для поддержания связи с любым другим, купившим гарнитуру, предателем."
+  - "Если вы предатель, то пингвины-гренадёры не будут вас атаковать, и будут взрываться при смерти."
+  - "Будучи ядерным оперативником, держитесь в группе! Пусть ваша броня и крепка, ваши соратники гораздо лучше справятся с вашим спасением: и огнём прикроют, и в укрытие лежащего тебя оттащат."
+  - "Будучи ядерным оперативником помните, что связь имеет ключевое значение! Используйте свою рацию, чтобы поддерживать связь с вашими соратниками и координировать действия."
+  - "Вы можете осмотреть свою гарнитуру, чтобы узнать, какие радиоканалы вам доступны и как по ним говорить."
+  - "Будучи утилизатором всегда носите с собой GPS и записывайте координаты станции на случай, если ваш обломок затеряется в космосе."
+  - "Будучи барменом помните, что вы можете воспользоваться циркулярной пилой на своем ружье, чтобы его стало легче хранить."
+  - "Будучи клоуном, разнообразьте свои приколы! Никто не любит шутников с одной шуткой."
+  - "Будучи клоуном помните, что даже если вы потеряли свою банановую кожурку, то на вашем КПК всё ещё можно поскользнуться! Хонк!"
+  - "Будучи мимом помните, что ваш обет молчания - источник вашей силы. Нарушив его, вы лишитесь своих сил и чести."
+  - "Будучи адвокатом, если наказание кажется вам слишком суровым, попытайтесь обсудить это со смотрителем."
+  - "Будучи сотрудником службы безопасности, общайтесь и координируйте свои действия с коллегами при помощи радиоканала службы безопасности, чтобы избежать неразберихи."
+  - "Будучи атмосферным техником помните, что ваш атмос голопроектор блокирует газы, но при этом пропускает предметы. С его помощью вы можете быстро локализовать утечки газа, пожары и пробоины корпуса."
+  - "Будучи инженером помните, что вы можете ремонтировать окна сварочным аппаратом, находясь не в боевом режиме."
+  - "Будучи инженером помните, что вы можете электрифицировать решётку, проложив под ней электрический кабель."
+  - "Будучи капитаном, вы являетесь одной из самых приоритетных целей на станции. Всё — от революций и ядерных оперативников, до предателей, которым захотелось лишить вас вашего уникального лазерного пистолета или вашей жизни — всё это повод для беспокойства."
+  - "Будучи капитаном, всегда носите с собой диск ядерной аутентификации и поисковый навигатор. Неплохим вариантом будет передать одно из них другому главе, которому вы можете доверить их сохранность."
```

поисковый навигатор - один - его
```suggestion
  - "Будучи капитаном, всегда носите с собой диск ядерной аутентификации и поисковый навигатор. Неплохим вариантом будет передать один из них другому главе, которому вы можете доверить его сохранность."
```